src/main/java/Duke.java

```diff
@@ -1,10 +1,146 @@
+import java.util.*;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        ArrayList<Task> lst = new ArrayList<Task>();
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Hey! I'm Duke!");
+        System.out.println("What can I help you with?");
+        boolean done = false;
+        int tasks = 0;
+        while (done != true) {
+            String output = sc.nextLine();
+            if (validationInput(output)) {
+                if (output.equals("bye")) {
+                    System.out.println("Bye! See you around:)");
+                    done = true;
+                } else if (output.equals("list")) {
+                    int counter = 1;
+                    System.out.println("    ____________________________________________________________");
```

Comment:
Too many line printing can be distracting. You may want to consider defining a method that wraps around the contents printed. eg. printFormat("Hello im duke") prints '__________\nHello im duke\n__________'